### DISCUSSION TITLE
vue-devtools not working with Electron

### DISCUSSION THREAD

**Discussion by alitalaghat** [quasarframework/quasar#11119] [CLI - Electron mode] [answered]
I am developing an Electron app using Quasar v2.2 & @quasar/cli v1.2.1

I tried to use vue-devtools, but it did not detect the components. Please check the following screenshots

![devtools-a](https://user-images.githubusercontent.com/8073213/138918121-71242ad0-5d2c-4d35-8976-31dd10bbfce8.jpg)

![devtools-b](https://user-images.githubusercontent.com/8073213/138918126-faa70912-7251-4aad-bdcc-e71f51e00a47.jpg)

As note I use the following cmd for testing:
``` quasar dev -m electron --devtools```

**Comment by MilosPaunovic** [comment]
If you're developing with Quasar 2.x, that means your project is using Vue 3.x.

Devtools screenshot you've pasted says `Detected Vue 2.6.10`. Have you installed the `beta` version of DT which work with `Vue3`?
https://chrome.google.com/webstore/detail/vuejs-devtools/ljjemllljcmogpfapbkkighbhhppjdbg?hl=en

**Reply by alitalaghat** [reply]
Yes, I use Quasar 2 and the project must use Vue 3, But as you see the devtools shows Vue v2!
Regarding the beta chrome extension, Is it needed for developing Electron? I think in my environment it depends on @vue/devtools.

**Comment by alitalaghat** [comment, marked as answer]
The problem was the vue-devtools version. For Quasar v2 (VueJs v3), we need vue-devtools v6. 
```yarn add --dev "@vue/devtools@6.0.0-beta.19"```
